drivers/periph_common/Kconfig

```diff
@@ -4,6 +4,14 @@
 # General Public License v2.1. See the file LICENSE in the top level
 # directory for more details.
 
+menuconfig KCONFIG_MODULE_PERIPH_TIMER
+    bool "Configure timer peripheral"
+    depends on MODULE_PERIPH_TIMER
+    help
+        Configure Timer peripheral using Kconfig.
+
```

Comment:
It seems odd that a Timer menu is guarding the inclusion of a Kconfig that is meant to contain all peripheral configurations.

I suggest we replace this `menuconfig` for a general "Configure peripheral drivers". What do you think?

Reply:
Sure!